packages/teams-js/src/public/webStorage.ts

```diff
@@ -18,10 +16,6 @@ export namespace webStorage {
    */
   export function isWebStorageClearedOnUserLogOut(): boolean {
     ensureInitialized();
-    // return true as web storage is always cleared on desktop.
-    if (GlobalVars.hostClientType === HostClientType.desktop) {
```

Comment:
Synced up with @shrshindeMSFT  offline for this and he confirmed that webStorage isSupported() will always return true for HostClientType.desktop irrespective of the value set in `clientSupportedSDKVersion` in GlobalVars. So it is safe to remove this condition.